modules/installation-creating-aws-control-plane.adoc

```diff
@@ -149,9 +149,12 @@ directory. This value is the long string with the format
 <19> The IAM profile to associate with control plane nodes.
 <20> Specify the `MasterInstanceProfile` parameter value from the output of
 the CloudFormation template for the security group and roles.
-<21> The type of AWS instance to use for the control plane machines.
+<21> The type of AWS instance to use for the control plane machines based on your selected architecture.
 <22> The instance type value corresponds to the minimum resource requirements for
-control plane machines.
+control plane machines. For example `m6i.xlarge` is a type for `x86_64`
```

Comment:
In line 95, we can avoid giving default value as m6i.xlarge.  

> 
>   {
>     "ParameterKey": "MasterInstanceType", <21>
>     "ParameterValue": "m6i.xlarge" <22>
>   },

and make it:
```
  {
    "ParameterKey": "MasterInstanceType", <21>
    "ParameterValue": "" <22>
  },
```